.github/workflows/publish-draft-releases.yml

```diff
@@ -1129,6 +1133,8 @@ jobs:
             name: calamari
           -
             name: manta
+          -
+            name: dolphin
```

Comment:
You need to define some more output variables for the dolphin runtime like `calamari-runtime-current`, `manta-runtime-current`, etc